.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- Let's make sure the "test.md file" is ignored by git by put putting it in the gitignore file